mmd_tools/core/model.py

```diff
@@ -1552,7 +1497,7 @@ def __createNonCollisionConstraint(self, nonCollisionJointTable):
         rb = ncc_obj.rigid_body_constraint
         rb.disable_collisions = True
 
-        ncc_objs = bpyutils.duplicateObject(ncc_obj, total_len)
+        ncc_objs = duplicateObject(ncc_obj, total_len)
```

Comment:
I suggest using `FnContext.duplicate_object` instead of `bpyutils.duplicateObject`